contracts/upgradeMocks/Mock.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.18;
+
+ /* solhint-disable */
+contract Mock {
```

Comment:
let's maybe make the naming more clear of what this "Mock" is. specify that it's for the upgrade tests